tools/path-getter/path-getter.mjs

```diff
@@ -18,40 +18,54 @@
 import {execSync} from 'child_process';
 import {join} from 'path';
 
+import {Browser, computeSystemExecutablePath} from '@puppeteer/browsers';
+
 /**
  * Either return a value of `BROWSER_BIN` environment variable or gets the path
- * to the browser version defined in `.browser` file, downloading the required
- * version if needed.
+ * to the browser specified in `CHANNEL` environment variable if specified. If
+ * no channel or browser bin is provided in environment variable,
+ * returns defined in `.browser` file, downloading the required version if
+ * needed.
  * @return {string}
  */
 export function installAndGetChromePath() {
-  let BROWSER_BIN = process.env.BROWSER_BIN;
-  if (!BROWSER_BIN) {
-    BROWSER_BIN = execSync(
-      ['node', join('tools', 'install-browser.mjs')].join(' ')
-    )
+  if (process.env.BROWSER_BIN) return process.env.BROWSER_BIN;
```

Comment:
```suggestion
  if (process.env.BROWSER_BIN) {
    return process.env.BROWSER_BIN;
  }
```

Reply:
eslint and prettier allow this syntax

Reply:
I may be wrong but adding brackets is encouraged by Google style.